lib/peep.rb

```diff
@@ -0,0 +1,34 @@
+require 'PG'
+
+class Peep
+  attr_reader :peep_id, :peep, :user, :timestamp
+
+  def initialize(id:, peep:, user:, timestamp:)
+    @peep_id = id
+    @peep = peep 
+    @user = user
+    @timestamp = timestamp
+  end
+    
+  def self.add(peep, user)
+    results = DatabaseConnection.connection.exec_params("
+      INSERT INTO peeps (peep, user_fk, datetime) 
+      VALUES ($1, $2, CURRENT_TIMESTAMP) 
```

Comment:
what are the variables `$1` and `$2` ? aren't these global variables? since I didn't try the exercise i feel unqualified to comment but what was the reason that you used global variables?